source/user-manual/manager/configuring-cluster/advanced-settings.rst

```diff
@@ -1,27 +1,735 @@
+                    log_end_msg 0
+                    ;;
+                2)
+                    log_end_msg 1
+                    ;;
+                esac
+                exit $ret
+                ;;
+            reload|force-reload)
+                log_daemon_msg "Reloading haproxy" "${BASENAME}"
+                haproxy_reload
+                ret=$?
+                case "$ret" in
+                0|1)
+                    log_end_msg 0
+                    ;;
+                2)
+                    log_end_msg 1
+                    ;;
+                esac
+                exit $ret
+                ;;
+            restart)
+                log_daemon_msg "Restarting haproxy" "${BASENAME}"
+                haproxy_stop
+                haproxy_start
+                ret=$?
+                case "$ret" in
+                0)
+                    log_end_msg 0
+                    ;;
+                1)
+                    log_end_msg 1
+                    ;;
+                2)
+                    log_end_msg 1
+                    ;;
+                esac
+                exit $ret
+                ;;
+            status)
+                haproxy_status
+                ret=$?
+                case "$ret" in
+                0)
+                    echo "${BASENAME} is running."
+                    ;;
+                1)
+                    echo "${BASENAME} dead, but $PIDFILE exists."
+                    ;;
+                *)
+                    echo "${BASENAME} not running."
+                    ;;
+                esac
+                exit $ret
+                ;;
+            *)
+                echo "Usage: /etc/init.d/${BASENAME} {start|stop|reload|restart|status}"
+                exit 2
+                ;;
+            esac
+
+            :
+
+
+        .. raw:: html
+
+            </details>
+
+        And a :ref:`Configuration file <haproxy_configuration>` to get the service up and running.
+
+        1. It will be needed to put these files in the same directory and build the image
+
+        .. code-block:: console
+
+            # tree
+            .
+            ├── Dockerfile
+            ├── entrypoint.sh
+            ├── haproxy.cfg
+            └── haproxy-service
+
+        .. code-block:: console
+
+            # docker build --tag=haproxy-deploy .
+
+        2. After building the image can we run the haproxy service
+
+        .. code-block:: console
+
+            # docker run haproxy-deploy
+            TCPLOG: true HTTPLOG: true
+            * Starting haproxy haproxy                                                                                                                                                                                          [NOTICE]   (33) : haproxy version is 2.8.9-1842fd0
+            [NOTICE]   (33) : path to executable is /usr/sbin/haproxy
+            [ALERT]    (33) : config : parsing [/etc/haproxy/haproxy.cfg:3] : 'pidfile' already specified. Continuing.
```

Comment:
This was misaligned:
```suggestion
            # docker run haproxy-deploy
            TCPLOG: true HTTPLOG: true
            * Starting haproxy haproxy
            [NOTICE]   (33) : haproxy version is 2.8.9-1842fd0
            [NOTICE]   (33) : path to executable is /usr/sbin/haproxy
            [ALERT]    (33) : config : parsing [/etc/haproxy/haproxy.cfg:3] : 'pidfile' already specified. Continuing.
```

![image](https://github.com/wazuh/wazuh-documentation/assets/23361101/24d1856a-a2cf-46ce-a13a-58136ef5175b)